packages/jest-haste-map/src/types.js

```diff
@@ -23,3 +24,11 @@ export type WorkerMetadata = {
   dependencies: ?Array<string>,
 };
 export type WorkerCallback = (error: ?Error, metaData: ?WorkerMetadata) => void;
+
+export type CrawlerOptions = {
```

Comment:
> When I want to add a field, I add it to the type and then Flow tells me exactly where I need to add the field

Wouldn’t that also be the case if you added a field to a type that allows overspecifying?